brave/browser-laptop
# Allowing publisher toggle to include a site outside of the minimum payment requirements

Fixes #13879 

Simply put, this allows the publisher toggle button to act as a mechanism of inclusion for a site that does not yet meet the minimum criteria.

## Submitter Checklist:


## Test Plan:
**Test plan will update as work progresses**
1. Enable Brave Payments
2. Set minimums rather high (Ex: 1 minute visit time with 5 minimum visits)
3. Navigate to a web page.
4. Click on the Publisher Toggle button.
5. Confirm that the button 'enables', and that the web page has been added to the ledger with the minimum criteria.
6. Confirm that the site can be unpinned/pinned by further interacting with the button.

## Reviewer Checklist:


Tests
